examples/automm/object_detection/detection_train.py

```diff
@@ -0,0 +1,100 @@
+"""
+The example to train an object detection model in AutoMM.
+
+An example to finetune an MMDetection model on COCO:
+    python detection_train.py \
+        --train_path coco17/annotations/instances_train2017.json
```

Comment:
So it will be like build a custom coco dataset like [this (using code)](https://www.immersivelimit.com/tutorials/create-coco-annotations-from-scratch) or [this (using DataTorch GUI)](https://anujsyal.com/building-computer-vision-datasets-in-coco-format)?

Reply:
I think it will be more like the first one using code. Let's see how users respond. If we need GUI later, let's decide at that time which GUI to use.

Reply:
Got it, I can add this tutorial first after multi gpu. Or do you think this has higher priority?

Reply:
Not high priority, as long as it is done before release 0.6.